### PR TITLE
mj-text > added letter spacing & updated readme

### PR DESCRIPTION
**Pull request by jitendravyas**

### CODE REVIEW

File: packages/mjml-text/src/index.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ class Text extends Component {
         fontSize: defaultUnit(mjAttribute('font-size'), "px"),
         fontStyle: mjAttribute('font-style'),
         fontWeight: mjAttribute('font-weight'),
+        letterSpacing: mjAttribute('letter-spacing', "px")
```
Comment (iRyusa) [inline packages/mjml-text/src/index.js]: Missing `, `

**Comment by blaugueux** [comment]
+1

**Comment by shulcsm** [comment]
Bump, can we fix the comma and have this merged?

File: packages/mjml-text/src/index.js (hunk 1 above)
Comment (iRyusa) [inline packages/mjml-text/src/index.js]: `defaultUnit(mjAttribute('letter-spacing'), "px"),` here instead

**Comment by iRyusa** [comment]
@jitendravyas Same here, closing this one because no real update for a while now. 

@shulcsm @blaugueux or @jitendravyas feel free to open a new PR with it !

**Comment by blaugueux** [comment]
@iRyusa https://github.com/mjmlio/mjml/pull/475

**Pull request closed without merging**